README.md

```diff
@@ -50,3 +50,13 @@ Cite from: <https://github.com/standard/eslint-config-standard/blob/v10.2.1/esli
 If your project conflicts with that configurations, please overwrite them.
 
 [eslint-config-standard]: https://github.com/standard/eslint-config-standard "ESLint Config for JavaScript Standard Style."
+
+## Semantic Versioning Guidelines
+
+- When bumping eslint or base rule, bump corresponding version part of this module.
```

Comment:
[nits] ESLint is a proper noun. I think that I should follow the manufacturer's notation. Like "**ESL**int"